src/test/it/bom.yml

```diff
@@ -0,0 +1,55 @@
+metadata:
+  # labels and annotations are key: value string attributes
+  labels:
+    name: bom-it
+    # Needed to make PCT happy
+    groupId: org.jenkins-ci.main
+    artifactId: jenkins-war
+    version: 2.121-artifact-manager-s3-SNAPSHOT
+spec:
+  core:
+    version: 2.121
+  plugins:
+    - groupId: "org.jenkins-ci.plugins.workflow"
+      artifactId: "workflow-api"
+      version:  2.28-rc337.8abe7c5204d9
+    - groupId: "org.jenkins-ci.plugins"
+      artifactId: "copyartifact"
+      version: 1.40-rc298.60da56cfabea
+    # Maven HPI Plugin improperly resolves dependency trees, needs upper bounds check
+    - groupId: "org.jenkins-ci.plugins"
+      artifactId: "structs"
+      version: 1.14
+    - groupId: "org.jenkins-ci.plugins.workflow"
+      artifactId: "workflow-step-api"
+      version: 2.15-rc308.607bdefd8c6a
+    - groupId: "org.jenkins-ci.plugins.workflow"
+      artifactId: "workflow-job"
+      version: 2.20
+    - groupId: "org.jenkins-ci.plugins.workflow"
+      artifactId: "workflow-basic-steps"
+      version: 2.8-rc351.c6608322f479
```

Comment:
can be removed, because it comes from dependencies (need to solve upper bounds properly tho) => APPEND pom.xml deps here unless overwritten?

Reply:
_All_ of the `plugins` would better have their version be inherited from `pom.xml`. For that matter the whole section could be deleted I think and Maven test-scoped classpath used (with `copyartifact` added if this is in fact in active use).

Reply:
There is a task for that: https://issues.jenkins-ci.org/browse/JENKINS-51574
But  cannot commit on any ETA